### ISSUE TITLE
Audit logs writing to ES index fail

### ISSUE THREAD

**Issue by kiblik** [sscarduzio/elasticsearch-readonlyrest-plugin#610]
Hi.

I setup ROR plugin to log audit logs to ES index.

```
readonlyrest:
    access_control_rules:
        ...
    audit_collector: true
    audit_index_template: '''audit''-yyyy-MM-dd'
    ldap:
        ...
```

ROR created index but it is empty all time. I found also this error in logs:
```
...
[2020-06-30T12:17:30,646][ERROR][t.b.r.e.p.EsAuditSink    ] [syselk1-mgm.st.sk] Some failures flushing the BulkProcessor: 
[2020-06-30T12:17:30,646][ERROR][t.b.r.e.p.EsAuditSink    ] [syselk1-mgm.st.sk] 13x: java.lang.IllegalArgumentException: Rejecting mapping update to [audit-2020-06-30] as the final mapping would have more than 1 type: [_doc, ror_audit_evt]
...
```

OS: CentOS-7
ES: 7.7.0
ROR: 1.19.5

Thanks for help.

**Comment by sscarduzio** [comment]
This is fixed in latest ror 1.20.0

**Comment by kiblik** [comment]
It works. Thanks
